benchmarks/src/test/scala/io/joern/benchmarks/testfixtures/BenchmarkFixture.scala

```diff
@@ -30,7 +30,7 @@ class BenchmarkFixture(
     with Matchers
     with BeforeAndAfterAll {
 
-  val semanticsFile: String = ProjectRoot.relativise("dataflowengineoss/src/test/resources/default.semantics")
+  val semanticsFile: String = ProjectRoot.relativise("benchmarks/src/test/resources/default.semantics")
```

Comment:
Did you forget to add that file? It doesn't seem to exist...

Reply:
Ah yes thanks my bad